WcaOnRails/config/locales/en.yml

```diff
@@ -365,6 +365,18 @@ en:
         guests_enabled:
           "true": "Ask about guests"
           "false": "Do NOT ask about guests"
+  mail_form:
```

Comment:
Check out [this comment](https://github.com/thewca/worldcubeassociation.org/pull/1390#discussion_r104502479). I did a rebase at some point, so that the comment got collapsed and I guess you didn't notice it.